modules/enet/networked_multiplayer_enet.cpp

```diff
@@ -496,7 +510,7 @@ uint32_t NetworkedMultiplayerENet::_gen_unique_id() const {
 					(uint32_t)OS::get_singleton()->get_unique_ID().hash64(), hash );
 		*/
 		hash = hash_djb2_one_32(
-				(uint32_t)((uint64_t)this), hash); //rely on aslr heap
+				(uint32_t)((uint64_t) this), hash); //rely on aslr heap
```

Comment:
Weird. Might be due to using different clang-format versions, which one do you have? Travis is using 3.9.